src/dlr_common.cc

```diff
@@ -1,7 +1,7 @@
-#include "dlr_common.h"
```

Comment:
why this change is needed?

Reply:
not needed something my editor did i guess. Shouldn't cause any harm

Reply:
We use `cmake-format --style=Google` to format C/C++ files for consistency.

Reply:
Return `#include "dlr_common.h"` on top.